packages/components/src/__actions__/LinkButton/v3/_docs/LinkButton--api-specification.mdx

```diff
@@ -0,0 +1,281 @@
+import { Canvas, Meta, Controls, ArgTypes, DocsStory } from '@storybook/blocks'
+import { ResourceLinks, KAIOInstallation, LinkTo } from '~storybook/components'
+import * as exampleStories from './LinkButton.doc.stories'
+
+<Meta title="Actions/LinkButton/LinkButton (v3)/API Specification" />
```

Comment:
There's no more Actions group now, so you'll want to move all the stories back to Components :)

Reply:
Cool I think I've moved everything back ok 👍 ready for a final round 🔔💥 🥊